compiler/luci-interpreter/src/loader/nodes/Conv2D.cpp

```diff
@@ -35,8 +35,10 @@ std::unique_ptr<Kernel> build_kernel_CircleConv2D(const luci::CircleNode *circle
   const Tensor *bias = helper.getOptionalInputTensor(node->bias());
   Tensor *output = helper.getOutputTensor(node);
 
+  // It is unknown what data will be stored in scratchpad tensor,
+  // using SINT8 as a most general option
```

Comment:
Comment is redundant and invalid for now.

Reply:
Better to remove =)

Reply:
see https://github.com/Samsung/ONE/pull/8069#discussion_r759210782